1-controllers/articles-controller.js

```diff
@@ -0,0 +1,25 @@
+const { selectArticles, selectArticleById } = require("../1-models/articles-model")
+
+
+exports.getArticles = (req, res, next) => {
+    selectArticles()
+    .then((articles) => {
+        res.status(200).send({ articles })
+    })
+    .catch(next)
+}
+
+exports.getArticleById = (req, res, next) => {    
+    const { article_id } = req.params
+   
+    if (isNaN(article_id) || article_id <= 0) {
```

Comment:
PSQL is designed to kick back an error code if the wrong data type is used, you shouldn't be using conditional logic here for this purpose.